backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -61,5 +64,44 @@ package object backend {
 
   case class PreemptedException(msg: String) extends Exception(msg)
 
+  /**
+    * @param usedInCallCaching Whether or not this attribute is used to determine a cache hit or miss. ALL attributes are hashed and stored.
+    */
   case class RuntimeAttributeDefinition(name: String, required: Boolean, default: Option[WdlValue], usedInCallCaching: Boolean)
+
+  object RuntimeAttributeDefinition {
```

Comment:
I can anticipate a complaint about the siting of this code in the package object